install_api.sh

```diff
@@ -415,10 +415,13 @@ setup_api() {
     fi
     exec_cmd "chown -R root:ossec $API_PATH/node_modules"
     exec_cmd "chmod -R 750 $API_PATH/node_modules"
-
-    if [ ! -f $API_PATH/configuration/auth/htpasswd ]; then
-        exec_cmd "ln -s $API_PATH/node_modules/htpasswd/bin/htpasswd $API_PATH/configuration/auth/htpasswd"
+    
+    # it is necessary to remove htpasswd in order to correctly create the symbolic link
+    if [ -e $API_PATH/configuration/auth/htpasswd ]; then
+        exec_cmd "rm $API_PATH/configuration/auth/htpasswd"
     fi
+        
+    exec_cmd "ln -s $API_PATH/node_modules/htpasswd/bin/htpasswd $API_PATH/configuration/auth/htpasswd"
```

Comment:
In addition, you can avoid the check and the `rm` lines if you add the `-f` option to `ln`. This will overwrite the destination file:

`ln -fs $API_PATH/node_modules/htpasswd/bin/htpasswd $API_PATH/configuration/auth/htpasswd`